app/src/test/java/com/hedvig/app/ComputedValuesTest.kt

```diff
@@ -62,6 +62,13 @@ class ComputedValuesTest {
         assertThat(result).isEqualTo("1337")
     }
 
+    @Test
+    fun `test remove quote from string`() {
```

Comment:
I think this test could be named slightly more descriptively 😉 

```suggestion
    fun `should evaluate string constants correctly`() {
```